### PR TITLE
Update create max spend fee estimation

### PR DESCRIPTION
**Pull request by lazynina**

### CODE REVIEW

**Comment by lazynina** [comment]
Current dependencies on/for this PR:
* feature/proof-of-stake
  * **PR #826** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/826?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #867** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/867?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #868** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/868?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #869** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/869?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #870** <a href="https://app.graphite.dev/github/pr/deso-protocol/core/870?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/deso-protocol/core/870?utm_source=stack-comment).

File: lib/blockchain.go [hunk 1]
```diff
@@ -4906,7 +4906,17 @@ func (bc *Blockchain) CreateMaxSpend(
 		prevFeeAmountNanos := uint64(0)
 		for feeAmountNanos == 0 || feeAmountNanos != prevFeeAmountNanos {
 			prevFeeAmountNanos = feeAmountNanos
-			feeAmountNanos = _computeMaxTxV1Fee(txn, minFeeRateNanosPerKB)
+			if bc.BlockTip().Height >= bc.params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight &&
+				feeEstimator != nil {
+				// TODO: replace MaxBasisPoints and 100*100 with variables configured by flags.
+				feeAmountNanos, err = feeEstimator.EstimateFee(txn, MaxBasisPoints, 100*100,
```
Comment (tholonious) [inline lib/blockchain.go:4912]: May as well add a PR that creates a const for the 100*100 now, and then switch it to a flag later. This way, we don't have the 100*100 sprinkled across the codebase now

File: lib/blockchain.go [hunk 2]
```diff
@@ -5031,7 +5041,9 @@ func (bc *Blockchain) AddInputsAndChangeToTransactionWithSubsidy(
 		txArg.TxnFeeNanos = 0
 		if txArg.TxnMeta.GetTxnType() != TxnTypeBlockReward {
 			if blockHeight >= bc.params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight && feeEstimator != nil {
-				txArg.TxnFeeNanos, err = feeEstimator.EstimateFee(txArg, MaxBasisPoints, 100*100, bc.params.MaxBlockSizeBytes)
+				// TODO: replace MaxBasisPoints and 100*100 with variables configured by flags.
+				txArg.TxnFeeNanos, err = feeEstimator.EstimateFee(txArg, MaxBasisPoints, 100*100,
```
Comment (tholonious) [inline lib/blockchain.go:5045]: ditto

**Review by tholonious — APPROVED** [review]

File: lib/blockchain.go (hunk 1 above)
Comment (lazynina) [reply lib/blockchain.go:4912]: We'll just use `MaxBasisPoints` for now.

**Pull request merged**